test/cmd/apps.sh

```diff
@@ -440,6 +440,40 @@ run_deployment_tests() {
   kubectl delete configmap test-set-env-config "${kube_flags[@]:?}"
   kubectl delete secret test-set-env-secret "${kube_flags[@]:?}"
 
+  ### Get rollout history
```

Comment:
I guess, it is difficult to add test cases also for statefulset and daemonsets because of their nature?

Reply:
It shouldn't be.  Let me see if I can add some